go-controller/pkg/metrics/master.go

```diff
@@ -143,6 +143,29 @@ var metricV6AllocatedHostSubnetCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of v6 host subnets currently allocated",
 })
 
+var MetricNetworkPolicyCountInNameSpace = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "num_network_policies_in_namespace",
+	Help: "A metric that captures the number of network policies in a namespace. This metric is labled with " +
+		"namespace name."},
+	[]string{
+		"namespace",
+	},
+)
+
+var MetricPodCountInNetworkPolicy = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "pod_count_in_policy",
+	Help: "A metric that captures the number of affected pods in a network policy. This metric is labled with " +
```

Comment:
"affected" is ambiguous in this sense. Do you mean pods to which the policy is applied, or pods that are selected as an ingress or egress peer?

Reply:
Thanks. Changed the description a bit.